server/package.json

```diff
@@ -59,11 +59,12 @@
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "cookie-parser": "^1.4.6",
-    "exiftool-vendored": "^19.0.0",
+    "exiftool-vendored": "^22.0.0",
```

Comment:
Do we need to update `exiftool-vendored` in this PR?

Reply:
Yes, we need to update it as ProjectionType does not exist in previous version